boot/bootstate20.go

```diff
@@ -754,3 +754,27 @@ func trustedCommandLineBootState(dev Device) *bootState20CommandLine {
 		dev: dev,
 	}
 }
+
+// bootState20RecoverySystem implements the successfulBootState interface for
+// tried recovery systems
+type bootState20RecoverySystem struct {
+	dev Device
+}
+
+func (ba20 *bootState20RecoverySystem) markSuccessful(update bootStateUpdate) (bootStateUpdate, error) {
```

Comment:
```suggestion
func (brs20 *bootState20RecoverySystem) markSuccessful(update bootStateUpdate) (bootStateUpdate, error) {
```

?

but again the acronym short hands for all the bootstate things are getting a bit unreadable anyways so maybe it doesn't matter so much

Reply:
I think ba20 comes from bootState20BootAssets.  So yes brs20  here,  and bcl20  for the commandline previously landed would be better.